### ISSUE TITLE
[Enhancement] Disable looping of CarouselView when only 1 item

### ISSUE THREAD

**Issue by Alex-Dobrynin** [xamarin/Xamarin.Forms#13442]
Right now, if loop property set to true and I have only 1 item in CarouselView I can scroll it and this item will appear again and again.

My suggestion is to disable looping when only 1 item in CarouselView and loop set ot true

**Comment by PawKanarek** [comment]
I think, that CaroulelView works good here, there might be case where developer/UI want to scroll just 1 item infinity times.
IMO you should set Loop property to false by yourself when you got just 1 item. That won't be hard.

**Comment by rmarinho** [comment]
@Alex-Dobrynin i don't think that might be needed, since the user should be able to do that.  Just listen to the items in the collection and disable the loop.

**Comment by Qythyx** [comment]
I tried binding `Loop` to a bool that does exactly what was suggested, but it seems that `CarouselView` does not listen for that binding to change. It does detect it the first time the form is loaded, but when the binding value is changed (and, yes, I do raise `OnPropertyChanged`) it does not detect the change.

**Comment by alexandru-w98** [comment]
Is this problem solved? I'm facing it right now. And if it is solved can someone point me to the answer pls?

**Comment by Qythyx** [comment]
I did manage to get this working by binding `Loop` to a bool that disables looping when there is only 1 item in the collection.

**Comment by alexandru-w98** [comment]
Link to workaround: https://github.com/xamarin/Xamarin.Forms/issues/13706
The workaround from the given link works if I don't change the collection dimensions. Right now I have a collection of 3 elements bound to the CarouselView and I also have a button with a command bound to it. The code from the command is :

void OnChangeLoop(){
            Items.Clear();
            Items.Add("item1");
            Items.Add("item2");
            Loop = !Loop;
        }
and in the code behind I have the workaround with the PropertyChanged. And when I press the button I get java.Lang.IllegalArgumentException: 'Invalid target position
